scripts/smoke-test.js

```diff
@@ -9,9 +9,9 @@ const run = require('../cli/run');
 
 function replacePlaceholders(content) {
   return content
-    .replace(/dredd-hooks-\{\{mylanguage\}\}/g, 'dredd-hooks-python')
-    .replace(/import hooks/g, "import dredd_hooks as hooks")
-    .replace(/\.\{\{myextension\}\}/g, '.py');
+    .replace(/\{\{my\-executable\-path\}\}/g, 'dredd-hooks-python')
+    .replace(/import hooks/g, 'import dredd_hooks as hooks')
+    .replace(/\.\{\{my\-extension\}\}/g, '.py');
```

Comment:
Shouldn't we match then end of the string?

```js
/\.\{\{my\-extension\}\}$/g
```

Reply:
No, because it's used like this in the feature files:

```
When I run `dredd ./apiary.apib http://localhost:4567 --server="node server.js" --language="dredd-hooks-{{mylanguage}}" --hookfiles=./hookfile.{{myextension}} --loglevel=debug`
```

or like this:

```
Given a file named "hookfile.{{myextension}}" with:
```